src/electionguard_cli/steps/shared/election_record_step.py

```diff
@@ -43,8 +59,24 @@ def run(
                 decrypt_results.lagrange_coefficients,
                 election_record_directory=temp_dir,
             )
-            file_name = splitext(election_inputs.output_file)[0]
+            file_name = splitext(election_inputs.output_record)[0]
             make_archive(file_name, self._COMPRESSION_FORMAT, temp_dir)
-            echo(
-                f"Successfully exported election record to '{election_inputs.output_file}'"
-            )
+            echo(f"Exported election record to '{election_inputs.output_record}'")
+
+    def _export_private_keys(self, election_inputs: E2eInputs) -> None:
+        if election_inputs.output_keys is None:
+            return
+
+        private_guardian_records = [
+            guardian.export_private_data() for guardian in election_inputs.guardians
+        ]
+        file_path = election_inputs.output_keys
+        for private_guardian_record in private_guardian_records:
+            file_name = private_guardian_record.guardian_id
+            to_file(private_guardian_record, file_name, file_path)
+        echo(
+            f"Exported {len(private_guardian_records)} private guardian keys to '{election_inputs.output_keys}'"
+        )
+        self.print_warning(
+            f"The files in {file_path} can decrypt an entire election. Protect the folder securely and do not share it."
```

Comment:
The files in {file_path} are secret and should be protected securely and not shared publicly.